atc/db/pipeline.go

```diff
@@ -653,6 +654,40 @@ func (p *pipeline) Rename(name string) error {
 	return err
 }
 
+func (p *pipeline) Move(destinationTeamName string) error {
+	destinationTeamID, err := psql.Select("id").
```

Comment:
We should be checking for if an error occurred here!

Reply:
done